aspnetcore/signalr/client-features.md

```diff
@@ -12,19 +12,19 @@ uid: signalr/client-features
 
 ## Feature distribution
 
-The table below shows the features and support for the clients that offer real-time support.
+The table below shows the features and support for the clients that offer real-time support. For each feature, the *minimum* version supporting this feature is listed. If no version is listed, the feature is not supported.
 
 | Feature | .NET | JavaScript | Java |
 | ---- | :-: | :-: | :-: |
-| Azure SignalR Service Support |✔|✔|✔|
-| [Server-to-client Streaming](xref:signalr/streaming)          |✔|✔|✔|
-| [Client-to-server Streaming](xref:signalr/streaming)          |✔|✔|✔|
-| Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |✔|✔| |
-| WebSockets Transport |✔|✔|✔|
-| Server-Sent Events Transport |✔|✔| |
-| Long Polling Transport |✔|✔|✔|
-| JSON Hub Protocol |✔|✔|✔|
-| MessagePack Hub Protocol |✔|✔| |
+| Azure SignalR Service Support |1.0.0|1.0.0|1.0.0|
+| [Server-to-client Streaming](xref:signalr/streaming)          |1.0.0|1.0.0|1.0.0|
+| [Client-to-server Streaming](xref:signalr/streaming)          |3.0.0|3.0.0|3.0.0|
+| Automatic Reconnection ([.NET](/aspnet/core/signalr/dotnet-client?view=aspnetcore-3.0&tabs=visual-studio#handle-lost-connection), [JavaScript](/aspnet/core/signalr/javascript-client?view=aspnetcore-3.0#reconnect-clients))          |3.0.0|3.0.0|❌|
```

Comment:
Not sure about the ❌. It doesn't line up great with the version numbers in GitHub. We'll see how the final review version comes out. I can continue to leave it blank if we think that would be clearer.

Reply:
Looks like the alignment is OK, so I think we can keep it:

![image](https://user-images.githubusercontent.com/7574/65978735-16462a80-e429-11e9-8e6d-89eedc881df6.png)

Reply:
We Should probably change the `monikerRange: '>= aspnetcore-3.0'` up top.

Reply:
I **really** like it. thanks!